src/validation.cpp

```diff
@@ -740,6 +739,8 @@ class MemPoolAccept
         CTxMemPool::setEntries m_all_conflicts;
         /** Mempool transactions that were replaced. */
         std::list<CTransactionRef> m_replaced_transactions;
+        /* Changeset representing adding a transaction and removing its conflicts. */
```

Comment:
circa 27c2abffbd9c5a83fd2becf6d7b70c7b30031f76 it's applying multiple transactions
```Suggestion
        /* Changeset representing adding transactions and removing their conflicts. */
```

Reply:
Done